Source/msg.h

```diff
@@ -210,7 +210,7 @@ struct TCmdGItem {
 	uint8_t bLevel;
 	uint8_t x;
 	uint8_t y;
-	uint16_t wIndx;
+	_item_indexes wIndx;

```

Comment:
These structs get passed over the network as-is. Here it pays to have exact control over the fields' sizes, irrespective of what else is happening in the codebase. I'd leave the standard ints in place.

Reply:
_item_indexes is set to be backed up by uint16_t, using _item_indexes is actually better as it means that mods would only have to adjust the size of _item_indexes if they add more items to the game

Reply:
Do we want to support multiplayer:
 * across different versions of DevilutionX? (Probably not.) If so, we can't afford to change the message format.
 * across different architectures? If so, we need to do byte swaps to keep the endianness consistent. Then the enum loses its meaning in this context.

Reply:
The game already checks the version and refuses to join if the versions do not match.

Network endianness is being worked on, still needs some testing:
https://github.com/diasurgical/devilutionX/tree/endian